caps/models.py

```diff
@@ -68,6 +68,39 @@ def __str__(self):
     def get_absolute_url(self):
         return "/councils/%s/" % self.slug
 
+    @property
+    def powers(self):
+        powers = ['staff', 'spending']
+
+        if self.country != self.NORTHERN_IRELAND and self.authority_type != 'COMB' and self.authority_type != 'NMD':
+            powers.append('transport-planning')
+
+        if self.country != self.NORTHERN_IRELAND and self.authority_type == 'UA' or self.authority_type == 'MD' or self.authority_type == 'CTY':
+            powers.append('passenger-transport')
+
+        if self.country != self.NORTHERN_IRELAND and self.authority_type != 'COMB' and self.authority_type != 'NMD':
```

Comment:
Is there a reason not to merge all the if statements that are the same?

Reply:
I wonder if this might make more sense to do like
```
mapping = {
    'transport-planning': ['UA', 'CTY', 'CC', 'LBO']
    'passenger-transport: ['UA', 'MD', 'CTY']
...
}

for power, types in mapping.items():
    if self.authority_type in types:
        powers.append(power)

```

and then have a separate if for NI with the list for that.

Reply:
I thought about a map structure like you suggested, but was a bit worried we might soon want conditions that are more complex than just "council is one of these types". Felt like premature optimisation, given we’re not even sure this is how we want to store powers in the first place?